src/tools/cephfs/cephfs-shell

```diff
@@ -855,7 +855,7 @@ sub-directories, files')
             if index == 0:
                 self.poutput('{:25s}\t{:5s}\t{:15s}{:10s}{}'.format(
                     "1K-blocks", "Used", "Available", "Use%", "Stored on"))
-            if not is_dir_exists(i.d_name):
+            if is_dir_exists(i.d_name):
```

Comment:
wow, actually `is_dir_exists` is not a good check anyway. IT's listing all of the items in the directory until it finds a match and then checks the dentry is a directory. `is_dir_exists` should just be doing a stat!

Anyway, as for this line, it can be trivially simplified to:

`if i.is_dir():`

Can you try that?

Reply:
Yes, I have changed it.